src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  const maxDiscount = 50;
+  const discount = 20;
+
+  if (days < 3) {
+    return days * rent;
+  } else if (days < 7) {
+    return days * rent - discount;
+  } else {
+    return days * rent - maxDiscount;
```

Comment:
```suggestion
  } 
    return days * rent - maxDiscount;
```
No need in this `else` here